app/src/main/java/org/piramalswasthya/cho/repositories/UserRepo.kt

```
@@ -361,7 +373,7 @@ class UserRepo @Inject constructor(
                     }
                     Timber.d("Error Message $errorMessage")
                 }
-            } catch (e: retrofit2.HttpException) {
+            } catch (e: HttpException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)